integration-tests/cli/state-import.it.js

```diff
@@ -0,0 +1,131 @@
+import { oneLine } from 'common-tags'
+import StateImport from '@commercetools/state-importer'
+import { getCredentials } from '@commercetools/get-credentials'
+import { clearData } from './helpers/utils'
+import states from '../../packages/state-importer/test/helpers/sampleStates.json'
+
+let projectKey
+if (process.env.CI === 'true') projectKey = 'state-import-integration-test'
+else projectKey = process.env.npm_config_projectkey
+
+describe('State importer', () => {
+  let apiConfig
+  let stateImport
+
+  const logger = {
+    error: () => {},
+    warn: () => {},
+    info: () => {},
+    debug: () => {},
+  }
+
+  beforeAll(() =>
+    getCredentials(projectKey).then(credentials => {
+      apiConfig = {
+        host: 'https://auth.sphere.io',
+        apiUrl: 'https://api.sphere.io',
+        projectKey,
+        credentials,
+      }
+      // Clear all existing states
+      return clearData(apiConfig, 'states')
+    })
+  )
+
+  afterAll(() => clearData(apiConfig, 'states'))
+
+  beforeEach(() => {
+    stateImport = new StateImport({ apiConfig }, logger)
+  })
+
+  test('should create states on the CTP', async () => {
+    const reportMessage = oneLine`
+        Summary: there were 4 successfully imported states
+        (4 were newly created, 0 were updated and 0 were unchanged).`
+    const expected = {
+      reportMessage,
+      detailedSummary: {
+        created: 4,
+        updated: 0,
+        unchanged: 0,
+        createErrorCount: 0,
+        updateErrorCount: 0,
+        errors: [],
+      },
+    }
+    await stateImport.run(states)
+    expect(stateImport.summaryReport()).toEqual(
+      expect.objectContaining(expected)
+    )
+  })
+
+  test('should update states on the CTP', async () => {
+    const statesToUpdate = states.map(state => ({ ...state, initial: false }))
+    const reportMessage = oneLine`
+        Summary: there were 3 successfully imported states
+        (0 were newly created, 3 were updated and 1 were unchanged).`
+    const expected = {
+      reportMessage,
+      detailedSummary: {
+        created: 0,
+        updated: 3,
+        unchanged: 1,
+        createErrorCount: 0,
+        updateErrorCount: 0,
+        errors: [],
+      },
+    }
+    await stateImport.run(statesToUpdate)
```

Comment:
Just curious: is `run` the naming pattern? I've seen `execute` elsewhere.

Reply:
We use `run` for the exporters and `parse` for the parsers. The `execute` is used by the client